src/index.ts

```diff
@@ -1,13 +1,14 @@
+import { Request, Response } from 'express';
 const express = require('express');
 const { specs, swaggerUi } = require('../swagger');
```

Comment:
Let's convert all import to ES6 style as it more common for use with TypeScript.

```suggestion
import express, { Request, Response } from 'express';
import { serve } from 'swagger-ui-express';
import swagger from './swagger';
```